tests/ci/cdk/cdk/codebuild/github_ci_linux_x86_omnibus.yaml

```diff
@@ -149,3 +149,34 @@ batch:
         privileged-mode: true
         compute-type: BUILD_GENERAL1_LARGE
         image: AWS_ACCOUNT_ID_PLACEHOLDER.dkr.ecr.AWS_REGION_PLACEHOLDER.amazonaws.com/ECR_REPO_X86_PLACEHOLDER:ubuntu-19.10_clang-9x_sanitizer_latest
+
+    # When no SELECTCHECK env variable is undefined, formal verification is executed with a few parameters.
+    - identifier: ubuntu2004_clang10x_formal_verification_quickcheck
+      buildspec: ./tests/ci/codebuild/linux-x86/ubuntu-20.04_clang-10x_formal-verification.yml
+      env:
+        type: LINUX_CONTAINER
+        privileged-mode: false
+        compute-type: BUILD_GENERAL1_LARGE
```

Comment:
This doesn't need to be the same size as the other two formal verification targets?

Reply:
Added below comments in the file to explain the instance type selection.

```yaml
    # SAW does not support thread level parallelism.
    # Current benmarks show running quick check (using single process) can consume more than 7 GB memory. This high memory usage is normal and confirmed by Galois side.
    # Accoridngly, BUILD_GENERAL1_LARGE (8 vCPUs, 15 GB memory) is selected for quick check, which just verfies some simple parameters.
    # For select check (test with lots of parameters), BUILD_GENERAL1_2XLARGE (72 vCPUs, 145 GB memory) is used to speed up tests by creating multiple processes.
```